cmd/podman/images/pull.go

```diff
@@ -156,9 +156,21 @@ func imagePull(cmd *cobra.Command, args []string) error {
 		pullOptions.Username = creds.Username
 		pullOptions.Password = creds.Password
 	}
+
+	rawImgName := args[0]
+	// TODO: maybe need update at github.com/containers/image/v5/docker/docker_transport.go:79-81
+	// if use tag and digest, the tag should be ignored
+	if !strings.Contains(rawImgName, "://") {
```

Comment:
This is a loose, and _undocumented_, heuristic. What is the exact condition this is trying to implement? This is wrong for the two conditions I can think of.